python/dgl/dataloading/neighbor.py

```diff
@@ -152,3 +152,104 @@ class MultiLayerFullNeighborSampler(MultiLayerNeighborSampler):
     """
     def __init__(self, n_layers, return_eids=False):
         super().__init__([None] * n_layers, return_eids=return_eids)
+
+
+class MultiLayerEtypeNeighborSampler(BlockSampler):
```

Comment:
can we hide this kind of details from users? for example, users still only need to provide `MultiLayerNeighborSampler`. When the graph is `DistGraph`, we automatically sample a fixed number of neighbors for each relation type. A distributed heterogeneous graph in DistDGL always stores the edge type in the field of `dgl.ETYPE`. I don't think we need to ask users to specify it.

Reply:
Done.